geometry_msgs/msg/TransformStamped.msg

```diff
@@ -1,9 +1,13 @@
 # This expresses a transform from coordinate frame header.frame_id
-# to the coordinate frame child_frame_id
+# to the coordinate frame child_frame_id at the time of header.stamp
 #
 # This message is mostly used by the
-# <a href="http://www.ros.org/wiki/tf">tf</a> package.
+# <a href="http://www.ros.org/wiki/tf2">tf2</a> package.
 # See its documentation for more information.
+#
+# The child_frame_id is necessary in addition to the frame_id
+# in the Header to communication the full reference for the transform
```

Comment:
@tfoote 
```suggestion
# in the Header to communicate the full reference for the transform
```

Reply:
I do wonder if this paragraph isn't just redundant with the definition of transform and the first paragraph.

Reply:
It is somewhat redundant but people asked for clarification in the API review so I figured another sentence could help.